salt/state.py

```diff
@@ -2176,15 +2188,17 @@ def check_requisite(self, low, running, chunks, pre=False):
                 run_dict = self.pre
             else:
                 run_dict = running
+
+            while True:
+                if self.reconcile_procs(run_dict):
+                    break
+                time.sleep(0.01)
```

Comment:
Why this `time.sleep` is doing here?

Reply:
Ah, right. The `self.reconcile_procs` still will go through checks and pass through so the CPU still in load. But you still can use `0.0001`, which will down the CPU load, but will do faster loops.

Reply:
It seems we do a similar polling loop in call_chunks() that uses 0.01; since these will be polling longrunning processes I think it makes sense to stick with 0.01 (or perhaps change them both); but i think 10ms latency is sufficient. if someone needed more we should perhaps consider not polling/using a child watcher that tornado provides.

Reply:
updated to 0.0001